src/index.html

```diff
@@ -129,25 +129,13 @@ <h3>Build iOS, Android, Desktop, and Progressive Web Apps using React</h3>
           <h1 slot="title">My Name</h1>
           <div slot="author">
             <h2>My Name</h2>
-            <p>A bit about me/bio</p>
-            <div style="padding-top: 64px;">
-              <deckgo-social twitter="ionicframework"> 
-                <ion-icon alt="Me on Twitter" slot="icon" name="logo-twitter"></ion-icon>
-              </deckgo-social>
-              <deckgo-social linkedin="maxlynch">
-                <ion-icon alt="Me on Linkedin" slot="icon" name="logo-linkedin"></ion-icon>
-              </deckgo-social>
-              <deckgo-social full-url="https://ionicframework.com">
-                <ion-icon alt="My Blog" slot="icon" name="book-outline"></ion-icon>
-              </deckgo-social>
-              <deckgo-social dev="ionic">
-                <ion-icon alt="Me on DEV.to" slot="icon" name="code-outline"></ion-icon>
-              </deckgo-social>
-              <deckgo-social github="ionic-team">
-                <ion-icon alt="My GitHub" slot="icon" name="logo-github"></ion-icon>
-              </deckgo-social>
-            </div>
+            <p style="padding: 8px; margin: 0;">A bit about me/bio</p>
           </div>
+          <deckgo-social slot="social-link" twitter="ionicframework"><ion-icon alt="Me on Twitter" slot="icon" name="logo-twitter"></ion-icon></deckgo-social>

```

Comment:
For the social links, we want them to align one per "line" (see [current deployed version](https://present-react.ionicframework.com/) to see what I mean). Any way to flip these back?

Reply:
@dotNetkow sure, done (in such case the slots should not be provided).